dimes/dimes.py

```diff
@@ -74,10 +76,10 @@ def finalize_plot(self):
                         visible="legendonly"
                         if not time_series_subplot_pair.time_series.is_visible
                         else True,
-                        line=dict(
-                            color=time_series_subplot_pair.time_series.color,
-                            dash=time_series_subplot_pair.time_series.line_type,
-                        ),
+                        line={
+                            "color": time_series_subplot_pair.time_series.color,
+                            "dash": time_series_subplot_pair.time_series.line_type,
+                        },
```

Comment:
@nealkruis I didn't realize that {} and dict() are the same thing. Is it recommended to use {} instead of dict()?

Reply:
Yes. Pylint will give a warning if you use `dict()`.